### DISCUSSION TITLE
Standalone Border Router Data Transfer Capability Via Bluetooth?

### DISCUSSION THREAD

**Discussion by vindicatorr** [openthread/openthread#8595] [Ideas]
I'm not sure if this would be better to be presented to Matter, but I didn't see any "Discussions" option, but Bluetooth(BT) is used for commissioning, so I figured, why not try here...

I've been experimenting with my esp-c3 (802.11 and BT capable), and my nrf52840 (BT, and 802.15.4 capable).
I'm aware Espressif has their c6 in the pipeline with 802.11, 802.15.4, and BT.

But for those devices with bluetooth and 802.15.4, is something like BT 6lowpan going to be an option for communicating data with something like an Android phone or PC (via BT)?
I know commissioning can happen via BT. But the actual back/forth data?

I know I can tie my c3 and nrf52840 together as a co-processor, but until the c6 exists (or whatever Nordicsemi comes up with on their end similarly), it'd be nice to be able to use my nrf52840 as a standalone border router.
Having that capability, even while 802.11 exists, would still be saving power instead of using 802.11 I'd think.

I'm envisioning: Android phone/PC (via bluetooth chip-tool/home-assistant) -> nrf52840 (standalone, via Matter+Thread/BT(6lowpan)) -> wireless devices.
With c6, it seems like it'd be Matter+Thread/BT/802.11 to handle all possible wireless devices in production, unless I'm missing logistics/purpose/function.
I may be missing "reasons" why this wouldn't be feasible.

**Comment by jwhui** [comment]
Use of BLE in [Matter 1.0 Specification](https://csa-iot.org/developer-resource/specifications-download-request/) is limited to commissioning - operational communication over BLE is not supported.

**Reply by vindicatorr** [reply]
Yeah, I just found it odd that a low-power protocol (BT-LE) wouldn't be used for ops, but higher power 802.11 would.

I just came across [Home-Assistant's use of 802.11](https://www.seeedstudio.com/blog/2022/11/03/how-matter-will-transform-your-smart-home/) is purportedly being used with Matter. I haven't gone far into it just yet, and am setting up my esp32-c3 to do some testing. It still won't be communicating with any 802.15.4 stuff though since it doesn't have that radio.

It's supposedly "[experimental](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_matter.html)". Or maybe that's just Nordicsemi's specific implementation. But then my nrf52840 doesn't have wifi, but maybe some other product of theirs does like my esp32-c3 has BT+wifi, but no 802.15.4.

I don't foresee 802.15.4 being added to phones anytime soon, so all ops are going to have to go through higher-power wifi to a border router.

But again, this topic may be inappropriate here. I just wish Matter had a community "Discussions" page or something like OpenThread does.

I keep hopping around all of these places for these various components.
And while Zephyr doesn't seem to have anything regarding Matter, Matter has some references to Zephyr.
It's all a mess in my head right now as I'm just starting out with it.

I also found it odd that Matter publishes their code to github, but you have to register to download the spec? "Company Name"? I'm just a hobbyist looking to explore.
